### PR TITLE
Add failing access conditions to license parsing

### PR DESCRIPTION
**Pull request by alicefuzier**
Part of https://github.com/wellcometrust/platform/issues/4191

### CODE REVIEW

File: pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsData.scala [hunk 1]
```diff
@@ -57,19 +57,30 @@ case class MetsData(
...
-          accessCondition) || license.url.equals(accessCondition)
-      } match {
-        case Some(license) => Right(license)
-        case None =>
-          Left(new Exception(s"Couldn't match $accessCondition to a license"))
+      accessCondition.toLowerCase() match {
+        // A lot of METS record have "Copyright not cleared"
+        // or "rightsstatements.org/page/InC/1.0/?language=en" as dz access condition.
+        // They both need to be mapped to a InCopyright license so hardcoding here
+        case s if s == "copyright not cleared" => Right(License.InCopyright)
```
Comment (jamesgorrie) [inline pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsData.scala]: Is there a way to get a list of these to correct in the source data?

Or does the system already allow them to do that? It's definitely worth trying to fix in the source data too.

Comment (alicefuzier) [reply pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsData.scala]: I think the "Copyright not cleared" wording is not used anymore so these are old digitised records. If we hade a dashboard that links these logs to the bnumber we could easily supply them to the digitisation team but at the moment it involves trailing the logs and the queues and it's messy and complicated.
To be honest, I think this is incorrect data and these hardcoded fixes are not nice but they are the quickest way to get a clean run right now.
As I said yesterday, I think we should start working on proper visibility into these things, preferably before we add CALM

Comment (jamesgorrie) [reply pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsData.scala]: Yip- 💯 agreed that this is the right way to do it, just thinking of helping our pals in collections.

We should probably have the talk of observability within the pipeline soonish.

**Comment by alicefuzier** [comment]
Fixed the tests BTW

**Comment by alicefuzier** [comment]
I'm going to merge this as it a small change and has already been deployed in the new pipeline

**Pull request merged**